egs/librispeech/asr/simple_v1/model.py

```diff
@@ -0,0 +1,92 @@
+from torch import Tensor
+from torch import nn
+
+
+class Model(nn.Module):
```

Comment:
Does that correspond to one of standard Kaldi TDNN architectures? If yes, could you mention it somewhere?

Reply:
I dont think it quite does, it's very basic, not tuned in the least.  There is no subsampling, which we should do at some point, e. g. put `stride=3` in the middle somewhere.